packages/common/src/hooks/useTwitterButtonStatus.ts

```diff
@@ -14,24 +14,16 @@ export const useTwitterButtonStatus = (
     useState<ShareStatus>('idle')
 
   const userName = user?.name
-  const twitterHandle = user ? user.twitter_handle : null
+  const twitterHandle = user?.twitter_handle
 
   const additionalUserName = additionalUser?.name
-  const additionalTwitterHandle = additionalUser
-    ? additionalUser.twitter_handle
-    : null
-
-  // Initially twitter handle is undefined; after fetch it's
-  // set to either null or a value in `fetchUserSocials` sagas
-  const twitterHandleFetched =
-    twitterHandle !== undefined &&
-    (additionalUser ? additionalTwitterHandle !== undefined : true)
+  const additionalTwitterHandle = additionalUser?.twitter_handle ?? null
```

Comment:
this needs to wait for the dual read code to be removed right?

Reply:
yeah kept this pr up to help with other prs as it had all the changes. closing it. sorry for not making that clear earlier but thats why i didnt tag anyone.